src/ReachSets/ContinuousPost/TMJets/post.jl

```diff
@@ -1,6 +1,68 @@
 using TaylorSeries: set_variables
 using LazySets.Approximations: box_approximation
 
+import TaylorModels
+import IntervalArithmetic
+const IA = IntervalArithmetic
+const zeroI = IA.Interval(0.0) # [0.0, 0.0]
+const oneI = IA.Interval(1.0) # [1.0, 1.0]
+const symI = IA.Interval(-1.0, 1.0)
+@inline zeroBox(m) = IntervalBox(zeroI, m)
+@inline symBox(m) = IntervalBox(symI, m)
+
+function _to_hyperrectangle(tTM, xTM, n)
+    N = length(xTM)
+    SET_TYPE = Hyperrectangle{Float64, SVector{n, Float64}, SVector{n, Float64}}
+    Rsets = Vector{ReachSet{SET_TYPE}}(undef, N-1)
+    @inbounds for i in 1:N-1
+        Hi = convert(Hyperrectangle, xTM[i])
+        t0 = tTM[i]; t1 = tTM[i+1]
+        Rsets[i] = ReachSet(Hi, t0, t1)
+    end
+    return Rsets
+end
+
+function _to_intervalbox(tTM, xTM, n)
+    N = length(xTM)
+    SET_TYPE = IA.IntervalBox{n, Float64}
+    Rsets = Vector{ReachSet{SET_TYPE}}(undef, N-1)
+    @inbounds for i in 1:N-1
+        Bi = xTM[i]
+        t0 = tTM[i]; t1 = tTM[i+1]
+        Rsets[i] = ReachSet(Bi, t0, t1)
+    end
+    return Rsets
+end
+
+function _to_zonotope(tTM, vTM, n)
+    N = length(tTM)
+    SET_TYPE = Zonotope{Float64}
+    Rsets = Vector{ReachSet{SET_TYPE}}(undef, N-1)
+    @inbounds for i in 1:N-1
+        # pick the i-th Taylor model
+        X = vTM[:, i]
+
+        # pick the time domain of the given TM (same in all dimensions)
+        Δt = TaylorModels.domain(X[1])
+
+        # evaluate the Taylor model in time, the coefficents are now intervals
+        X_Δt = evaluate(X, Δt)
+
+        # builds the associated taylor model for each coordinate
+        X̂ = [TaylorModelN(X_Δt[k], X[k].rem, zeroBox(n), symBox(n)) for k in 1:n]
+
+        # floating point rigorous polynomial approximation
+        fX̂ = TaylorModels.fp_rpa.(X̂)
+
+        # LazySets can overapproximate a Taylor model with a Zonotope
+        Zi = overapproximate(fX̂, Zonotope)
```

Comment:
This seems to be the only place where `Zonotope` matters. Can we make this method parametric in any `LazySet`? (See also my other comment about the method.)

Reply:
What about this comment?

Reply:
> Can we make this method parametric in any LazySet?

Transforming a Taylor model to a lazyset can only be done ATM to zonotopes. It would be interesting to add a method for overapproximating with polytopes; [this notebook](https://github.com/mforets/escritoire/blob/master/sets/TaylorModels_SF.ipynb) has some ideas in that direction.

Reply:
But then why not add the possibility here so that it will automatically work after adding more support?

Reply:
The HPolyhedron needs the template directions, so it seems more appropriate to use <:AbstractDirections. (I created https://github.com/JuliaReach/LazySets.jl/issues/1787 as i didnt find an existing one). But, yes, the code for overapprox with a Polyhedron or other lazyset can re-use fX̂, the first part previous to line 58; what we do there is to bring vTM to a taylor model in space variables only, whose coefficients are intervals.